src/routes/gifs.js

```diff
@@ -104,17 +105,37 @@ const createComment = catchAsync( async (req, res) => {
 
 })
 
+const recordLikesOnGif = catchAsync( async(req, res) => {
+	const {id} = req.params
+	const { userId } = req.body
+	const newLike = await postService.recordPostLikes({
+		userId,
+		id
+		 })
+	res.status(201).json({
```

Comment:
not seeing us mapping the custom error in this file

Reply:
i don't get you please

Reply:
I get you now